src/components/modals/NumericInput.tsx

```diff
@@ -0,0 +1,38 @@
+import * as React from 'react'
+import { useRef } from 'react'
+import { TextInput, TextInputProps } from 'react-native'
+
+import { useHandler } from '../../hooks/useHandler'
+import { formatNumberInput, formatToNativeNumber, isValidInput } from '../../locales/intl'
+import { useState } from '../../types/reactHooks'
+
+type Props = {
+  ref: React.RefObject<TextInput>
```

Comment:
Do not require the ref as a prop like this. The forwardRef thing takes care of all the ref typings.

Reply:
Ack. I changed from accessing the `ref` as a prop to using `forwardRef` for forgot to remove the prop type. Will do